src/functions-and-arrays.js

```diff
@@ -1,38 +1,104 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1 = 0, num2 = 0) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num1 < num2) {
+    return num2;
+  } else {
+    return num1;
+  }
+}
 
+const highestNumber = maxOfTwoNumbers(12, 26);
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  let word = '';
+  if (arr === undefined) {
+    return null;
+  } else if (arr.length === 0) {
+    return null;
+  }
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i].length === 0) {
+      return null;
+    } else if (word.length < arr[i].length) {
+      word = arr[i];
+    } else if (word.length === arr[i].length) {
+      continue;
+    }
+  }
+  return word;
+}
 
+const longestWord = findLongestWord([]);
 
+console.log(longestWord);
```

Comment:
After you finish writing code, try to always clean up and remove any lines that are unused ;)